extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -93,6 +93,7 @@ export const defaultProjectNameStarter = localize('defaultProjectNameStarter', "
 export const location = localize('location', "Location");
 export const reloadProject = localize('reloadProject', "Would you like to reload your database project?");
 export const learnMore = localize('learnMore', "Learn More");
+export const sdkLearnMoreUrl = 'https://github.com/microsoft/DacFx/tree/main/src/Microsoft.Build.Sql';
```

Comment:
It would be good to update the README to explain a little more what it is - and even provide links to general docs for SDK style projects. Right now I'm not sure the information in the README is going to help users very much.

Reply:
@zijchen @dzsquared can we update the Microsoft.Build.Sql readme to have some more info to help someone unfamiliar with SDK style projects? Or if we have another page with more info I can update the link to go there.

Reply:
I do owe Drew a FAQ section on the README, I'll add more to it there